_posts/2015-09-25-idk.markdown

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: idk
+date: 2015-09-25 14:35:00
+homepage: https://github.com/UniFreak/unifreak.github.io
+download: https://codeload.github.com/UniFreak/unifreak.github.io/zip/master
+demo: https://unifreak.github.io
+author: FangHao
+thumbnail: idk.png
+license: MIT

```

Comment:
Please change the license value to `MIT License` so similar formatting is used for all of themes. Thanks!